wildfly-modules/jboss/container/wildfly/s2i-wildfly/bash/artifacts/usr/local/s2i/assemble

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
 export CONFIG_ADJUSTMENT_MODE=cli
-
-. "${JBOSS_CONTAINER_WILDFLY_S2I_MODULE}/assemble.sh"
\ No newline at end of file
+export MAVEN_ARGS="${MAVEN_ARGS} -Dcom.redhat.xpaas.repo.jbossorg"
```

Comment:
@luck3y , MAVEN_ARGS is not set in the image, we are relying on the cct_module default. If we are setting it here, cct_module default value will not get applied. I think that we need to append the property to MAVEN_ARGS_APPEND that the user can set and for which we don't have default values in cct_module.

Reply:
Just to add my two cents, notice MAVEN_ARGS_APPEND is by default set on the templates, so, in case of adding `-Dcom.redhat.xpaas.repo.jbossorg` in the scripts, it would be nice to take into account possible duplicates of the same property, bash property expansion facilities can help here. It would be harmless having the same property duplicated, but, well, just in case we want to avoid it.